### PR TITLE
2018 05 01

### PR DESCRIPTION
**Pull request by InfAmelia**

### CODE REVIEW

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 1]
```diff
@@ -227,14 +227,24 @@ Content Type: application/json
...
+
+      **"search_radius_miles"**: a number
+
+      **"restrict_to_state"**: Boolean. Default is to restrict search to be within state boundaries.
+
+      **"allocation_fallback_strategy"**: "UseTollFree" or "Wait". If we are unable to fill the RingPool using the location information provided, we will use this parameter to either fill it with TollFree ("UseTollFree") or continue to wait until more numbers become available. Default is "UseTollFree".
+
   * - tn_prefix_whitelist
     - array of strings
     - an array of stringified limiters on the boundaries of where to look for local numbers given as npa or npa nxx pairs (ex. ["805", "805612"])
```
Comment (peterwilson) [inline source/api_documentation/network_integration/ringpools/index.rst]: We're removing the functionality for npa nxx pairs in both the new and historical api versions. Can you update that info?

Comment (InfAmelia) [reply source/api_documentation/network_integration/ringpools/index.rst]: I'm assuming we need to update older docs as well?

Comment (peterwilson) [reply source/api_documentation/network_integration/ringpools/index.rst]: Yes, we'll want to change older docs.

File: source/api_documentation/network_integration/promo_numbers/index.rst [hunk 2]
```diff
@@ -98,17 +98,25 @@ When you POST a promo number, we return the promo_number in the response body. T
...
 
       **“state”**: a valid state
 
+  * - local_number_preferences
+    - hash
+    - Both are optional
+
+      **"search_radius_miles"**: a number
+
+      **"restrict_to_state"**: Boolean. Default is to restrict search to be within state boundaries.
```
Comment (peterwilson) [inline source/api_documentation/network_integration/promo_numbers/index.rst]: `allocation_fallback_strategy` is missing. It's optional. Default is `UseTollFree`. Other option is `Wait`.

File: source/api_documentation/network_integration/promo_numbers/index.rst [hunk 3]
```diff
@@ -98,17 +98,25 @@ When you POST a promo number, we return the promo_number in the response body. T
...
 
       **“city”**: a valid city
 
       **“state”**: a valid state
 
+  * - local_number_preferences
+    - hash
+    - Both are optional
+
+      **"search_radius_miles"**: a number
```
Comment (peterwilson) [inline source/api_documentation/network_integration/promo_numbers/index.rst]: Can we provide more info to the user here? 0 is acceptable and that it's the distance from the center of their search location. The default is <= 240. It must be an integer.

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 4]
```diff
@@ -227,17 +227,27 @@ Content Type: application/json
...
 
       **"latitude"**: a valid latitude
 
       **"longitude"**: a valid longitude
 
+  * - local_number_preferences
+    - hash
+    - All are optional
+
+      **"search_radius_miles"**: a number
```
Comment (peterwilson) [inline source/api_documentation/network_integration/ringpools/index.rst]: Can we provide more info to the user here? 0 is acceptable and that it's the distance from the center of their search location. The default is <= 240. It must be an integer.

File: source/api_documentation/network_integration/promo_numbers/index.rst (hunk 2 above)
Comment (peterwilson) [inline source/api_documentation/network_integration/promo_numbers/index.rst]: `Boolean. Default: True. Used to restrict search to be within state boundaries.`

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 5]
```diff
@@ -227,17 +227,27 @@ Content Type: application/json
...
+
+      **"restrict_to_state"**: Boolean. Default is to restrict search to be within state boundaries.
```
Comment (peterwilson) [inline source/api_documentation/network_integration/ringpools/index.rst]: `Boolean. Default: True. Used to restrict search to be within state boundaries.`

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 6]
```diff
@@ -227,17 +227,27 @@ Content Type: application/json
...
+
+      **"allocation_fallback_strategy"**: "UseTollFree" or "Wait". If we are unable to fill the RingPool using the location information provided, we will use this parameter to either fill it with TollFree ("UseTollFree") or continue to wait until more numbers become available. Default is "UseTollFree".
```
Comment (peterwilson) [inline source/api_documentation/network_integration/ringpools/index.rst]: I'd pull the default value to the front, just after the available choices.

**Review by peterwilson — CHANGES_REQUESTED** [review]
Let's add some more detail for the user about defaults and acceptable values.

File: source/api_documentation/network_integration/ringpools/index.rst [hunk 7]
```diff
@@ -227,17 +227,27 @@ Content Type: application/json
...
 
       **"latitude"**: a valid latitude
 
       **"longitude"**: a valid longitude
 
+  * - local_number_preferences
+    - hash
+    - All are optional
+
+      **"search_radius_miles"**: An integer, greater than or equal to 0, which represents the number of miles from the location to search. Default: 240
```
Comment (peterwilson) [inline source/api_documentation/network_integration/ringpools/index.rst]: Looking at the other two params, maybe we should say:
```
Integer, greater than or equal to 0. Default is 240. Represents the number of miles from the location to search.
```

**Review by peterwilson — COMMENTED** [review]
Maybe we should try to make the param descriptions more uniform. Thoughts?

**Review by peterwilson — APPROVED** [review]

**Comment by peterwilson** [comment]
Closing because we do not want to merge, we will publish this later and make this the master branch, per the README instructions.

**Pull request closed without merging**